query-node/schemas/membership.graphql

```diff
@@ -71,6 +71,9 @@ type Membership @entity {
   "Whether member is founding member."
   isFoundingMember: Boolean!
 
+  "Whether member is elected in the current council."
+  isCouncilMember: Boolean!
```

Comment:
This value is never set to `true`

Reply:
The sign is being set and unset properly now.